CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 Versions
 ========
+
+2022-23-06
+----------
+* Pass specific build_args for each arch
```

Comment:
This should be a bit more of a functional than technical description. Something around the lines of `Handle multi-arch build for all images` would probably be best. You can also detail the version bumps